src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -49,6 +49,7 @@
         flex: 1 1 auto;
         flex-direction: column;
         width: 100%; // Explicit width required since flex-direction is column
+        min-height: 93px;
```

Comment:
May be a good candidate for a CSS variable

Reply:
Thanks, I added a var